deutsche-gesellschaft-fur-psychologie.csl

```diff
@@ -270,7 +289,7 @@
             <text macro="container-contributors"/>
             <text macro="secondary-contributors"/>
             <group delimiter=", ">
-              <text variable="container-title" font-style="italic"/>
+              <text variable="container-title"/>
```

Comment:
as per the post on the forums, this has to be wrong, no? They still put containers in italics.